aioftp/server.py

```diff
@@ -1042,6 +1059,12 @@ def __init__(self,
 
                     connection.passive_server.close()
 
+                    if self.available_data_ports is not None:
+
+                        self.available_data_ports.put_nowait(
+                            (0, connection.passive_server_port)
+                        )
+
```

Comment:
Oh, I miss `if connection.future.passive_server.done()` since this was hidden. My bad.